cinema_service/settings.py

```diff
@@ -86,8 +87,11 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": BASE_DIR / "db.sqlite3",
+        "ENGINE": "django.db.backends.postgresql",
+        "HOST": os.environ["POSTGRES_HOST"],
```

Comment:
env names don't match. Did you run your project?

Reply:
Yes, buy env names in .env was different)